lib/src/robot_off_api_client.dart

```diff
@@ -4,15 +4,15 @@ import 'package:http/http.dart';
 
 import 'model/insight.dart';
 import 'model/robotoff_question.dart';
+import 'model/robotoff_question_order.dart';
```

Comment:
Strange that the lint check didn't complain before your PR. Or is the check only for "packages" and not for local files?

Reply:
Good question, maybe it's also because I use Dart 3.
And I know that some rules have changed